spec/system/nsm/disbursements_spec.rb

```diff
@@ -145,14 +193,15 @@
     fill_in 'Number of miles', with: 100
     fill_in 'Enter details of this disbursement', with: 'details'
 
+    expect(page).to have_content 'Do you need to add another disbursement?'
+    choose 'No'
+
     click_on 'Save and continue'
 
-    expect(page).to have_content 'Do you want to add another disbursement?'
+    expect(page).to have_no_content 'You cannot save and continue if any disbursements are incomplete'
```

Comment:
I think you want this expectation to happen after the final 'Save and continue'. At this current point you've just submitted the disbursement cost form, so _of course_ this message won't be here, because the disbursement cost form has no mechanism for adding that message. Whereas after line 204 below you've just submitted the DisbursementsForm which could generate this message but hopefully won't because there are now no incomplete disbursements. Do it's there that we want to verify that this message doesn't appear.